### ISSUE TITLE
CHANGELOG for v2 → v3 → v4 (list of breaking changes)

### ISSUE THREAD

**Issue by kachkaev** [jonschlinkert/set-value#41]
I see a number of warnings from `yarn audit` because some project dependencies still rely on `set-value` v2 and v3. While it’s technically possible to force-install v4 via `package.json` → [`resolutions`](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/#toc-how-to-use-it), it’s not clear whether that’s a save move or not.

It’d be great if the repo had `CHANGELOG.md`, `UPGRADING.md`, `MIGRATION.md` or info on the [Releases page](https://github.com/jonschlinkert/set-value/releases). If major versions are only related to supported Node versions or some rare edge cases, library users can make informed decisions to force-upgrade from v2 or v3.

Examples: `jest` → [CHANGELOG.md](https://github.com/facebook/jest/blob/main/CHANGELOG.md) & [Releases](https://github.com/facebook/jest/releases), `sentry-javascript` → [MIGRATION.md](https://github.com/getsentry/sentry-javascript/blob/master/MIGRATION.md) & [Releases](https://github.com/getsentry/sentry-javascript/releases)

Related to #38 & #39

**Comment by shayc** [comment]
Changes between 2.0.1 and 3.x: https://github.com/jonschlinkert/set-value/compare/2.0.1...3.0.0
Changes between 3.0.0 and 4.0.1: https://github.com/jonschlinkert/set-value/compare/3.0.0...4.0.1

History up to 3.0: https://github.com/jonschlinkert/set-value/blob/4.1.0/.verb.md#history 

TL;DR - Without reading too much into the source code, there has been a NodeJS bump, and ~maybe~ some API changes, not sure if they're breaking anything though.
